src/common/utility/configfile.cpp

```diff
@@ -39,6 +39,9 @@
 #include "configfile.h"
 #include "files.h"
 
+#include "filesystem.h"
```

Comment:
Is it for `FileExists()` function? If so, this function is defined in `cmdlib.h` file.